main.py

```diff
@@ -46,10 +46,10 @@ def createMsg(data):
     elif fileExtn == "zip" and fileType == "dd":
         msg["description"] = 'Data Delivery files uploaded to GCP bucket from Blaise5'
         msg["dataset"] = 'blaise_dde'
-        msg["iterationL1"] = os.getenv('ON-PREM-SUBFOLDER')
-        msg["iterationL2"] = data['name'][3:7].upper()
-        msg["iterationL3"] = data['name'][7:8].upper()
-        msg["iterationL4"] = ''
+        msg["iterationL4"] = 'SYSTEMS'
```

Comment:
Should this be iterationL4? There's now no iterationL1, and L4 twice?

Reply:
eek, well spotted